src/engine/ruleset/integrations/docker-listener/documentation.yml

```diff
@@ -0,0 +1,19 @@
+title: Wazuh Docker Listener Integration
+overview: >
+  This integration processes logs obtained with the docker-listener wodle from docker.
+  This are real-time events from the docker server.
+compatibility: >
+  This module has been tested against version 20.10.12
```

Comment:
specifies the version of the operating system and the version of the agent (since this is the one with woodle)